helm_deploy/values-dev.yaml

```diff
@@ -18,4 +18,5 @@ generic-service:
 # CloudPlatform AlertManager receiver to route promethues alerts to slack
 # See https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts
 generic-prometheus-alerts:
-  alertSeverity: syscon
+  sqsAlertsQueueNames:
+    - "Digital-Prison-Services-dev-hmpps_prisoner_to_nomis_dlq"
```

Comment:
This looks like the right fix, but I dont understand how alerts are already appearing allbeit in the default slack channel (dps_alerts_non_prod in this case) despite there being no sign of sqs alerts in:
% kc -n hmpps-prisoner-to-nomis-update-dev get PrometheusRules